crates/corro-agent/src/agent/tests.rs

```diff
@@ -412,15 +412,20 @@ pub async fn configurable_stress_test(
                     }
                 }
 
-                Ok::<_, eyre::Report>((*actor_id, statements.len()))
+                Ok::<_, eyre::Report>((*actor_id, 1))
             })
         }
     })
     .try_buffer_unordered(10)
-    .try_collect::<BTreeMap<_, _>>()
-    .await?;
+    .try_collect::<Vec<(ActorId, usize)>>().await?
+    .into_iter().fold(BTreeMap::new(), |mut acc, item| {
+        {
+            *acc.entry(item.0).or_insert(0) += item.1
+        }
+        acc
+    });
```

Comment:
```suggestion
    .try_fold(BTreeMap::new(), |mut acc, item| {
        {
            *acc.entry(item.0).or_insert(0) += item.1
        }
        Ok(acc)
    });
```

Reply:
I don't the amount of braces and derefs should be necessary here but I can't suggest more without seeing the compiler errors (happy to pair on this with you but it's not something particularly important)